src/20250814_Multi_GHOInkLaunch/GHOInkLaunchConstants.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title GHOInkLaunchConstants
+ * @notice Library containing extra constants needed across the GHO Ink Launch proposal
+ */
+library GHOInkLaunchConstants {
+  // Block Numbers for forking, below values to match /config.ts
+  uint256 internal constant INK_BLOCK_NUMBER = 22331165;
```

Comment:
I would put the block numbers directly on the test files as its the only place that will be used, and its confusing to have stuff from multiple networks, then followed by addresses of only the new network.